circuit_knitting/cutting/cutqc/__init__.py

```diff
@@ -33,3 +35,11 @@
     "create_dd_bin",
     "reconstruct_dd_full_distribution",
 ]
+
+warn(
+    f"The package {__name__} is deprecated and will be removed no sooner than Circuit Knitting Toolbox v0.8.0. "
```

Comment:
Didn't think of this until now, but why not have double backticks here to match the ones on the following line.

```suggestion
    f"The package ``{__name__}`` is deprecated and will be removed no sooner than Circuit Knitting Toolbox v0.8.0. "
```

Reply:
Oops, sorry, just made these changes after your review. You may have to review again :|